packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -21,7 +21,7 @@ describe('Dropdown', () => {
 
       wrapper.find({ className: Dropdown.slotClassNames.clearIndicator }).simulate('click')
       expect(onSelectedChange).toBeCalledTimes(1)
-      expect(onSelectedChange).toHaveBeenCalledWith(
+      expect(onSelectedChange).toHaveBeenLastCalledWith(
```

Comment:
Nit: I propose to use proper semantics there.
When we have multiple calls, let's use `toHaveBeenLastCalledWith()`, for calls let's use `toHaveBeenCalledWith()`